sql/arcfeedback.sql

```diff
@@ -0,0 +1,32 @@
+DROP TABLE IF EXISTS `arc_complaint_info`;
+
+CREATE TABLE `arc_complaint_info` (
```

Comment:
small nitpick of mine, can you line up the sql nicely? Also no need to surround everything with backticks.